core/jobs/src/main/java/com/cognifide/aet/job/common/modifiers/resolution/ResolutionModifier.java

```diff
@@ -72,26 +85,37 @@ public void setParameters(Map<String, String> params) throws ParametersException
       if (params.containsKey(HEIGHT_PARAM)) {
         height = NumberUtils.toInt(params.get(HEIGHT_PARAM));
         ParametersValidator
-            .checkRange(height, 1, MAX_SIZE, "Height should be greater than 0 and smaller than " + MAX_SIZE);
+            .checkRange(height, 1, MAX_SIZE,
+                "Height should be greater than 0 and smaller than " + MAX_SIZE);
+      } else {
+        samplingPeriod = Optional.ofNullable(params.get(SAMPLING_PERIOD_PARAM))
```

Comment:
There is no limit for the `samplingPeriod`.
Just a question here: shouldn't that be limited somehow?
What will happen if user sets here e.g. `10000000 ms`?

Reply:
You are correct, there should be a range for this param. I will set max at 10 seconds. Please give suggestion if the value seems reasonable.